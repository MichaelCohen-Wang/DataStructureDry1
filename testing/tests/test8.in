add_ship 170165 19
add_ship 1519966 5
remove_ship 1519966
get_cannons 1519966
add_ship 1467875 14
add_ship 344073 7
add_ship 95005 17
remove_ship 170165