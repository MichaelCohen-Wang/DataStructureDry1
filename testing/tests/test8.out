add_ship: SUCCESS
add_ship: SUCCESS
remove_ship: SUCCESS
get_cannons: FAILURE
add_ship: SUCCESS
add_ship: SUCCESS
add_ship: SUCCESS
remove_ship: SUCCESS
